hugegraph-llm/src/hugegraph_llm/operators/hugegraph_op/fetch_graph_data.py

```diff
@@ -28,6 +28,12 @@ def __init__(self, graph: PyHugeClient):
     def run(self, context: Optional[Dict[str, Any]]) -> Dict[str, Any]:
         if context is None:
             context = {}
+        if "num_vertices" not in context:
+            context["num_vertices"] = self.graph.gremlin().exec("g.V().id().count()")["data"]
+        if "num_edges" not in context:
+            context["num_edges"] = self.graph.gremlin().exec("g.E().id().count()")["data"]
         if "vertices" not in context:
             context["vertices"] = self.graph.gremlin().exec("g.V().id().limit(10000)")["data"]
+        if "edges" not in context:
+            context["edges"] = self.graph.gremlin().exec("g.E().id().limit(100)")["data"]
```

Comment:
seems we don't need to save all info in `context` (too broad to use it)

Consider a lifecycle that is as suitable/small as possible

Reply:
After analysis, it has been determined that the `context` in question is a local variable and is not the same as the `context` being passed globally. 
It has now been renamed to `graph_context`.